packages/suite-data/index.ts

```diff
@@ -20,13 +21,15 @@ const config = {
     'suite-native': [
         'fonts',
         'images',
+        'guide',
```

Comment:
nit: I would put guide under fonts, to be have it consistent 😆

Reply:
c73aa81f15a4a0b499fbbabb095fdcc0d7950537